frontend/components/FleetAce/FleetAce.tsx

```diff
@@ -70,16 +70,10 @@ const FleetAce = ({
   };
 
   const handleDelete = (deleteCommand: string) => {
-    const currentText = editorRef.current?.editor.getValue();
     const selectedText = editorRef.current?.editor.getSelectedText();
 
     if (selectedText) {
-      const remainingText = currentText?.replace(selectedText, "");
-      if (typeof remainingText !== "undefined") {
-        onChange && onChange(remainingText);
-        editorRef.current?.editor.navigateLeft();
-        editorRef.current?.editor.clearSelection();
-      }
```

Comment:
I'm not clear on precisely what this condition was doing, but it seems unnecessary with this new implementation. Let me know if you see a reason to keep this.